src/screens/Onboarding/steps/setupDevice/scenes/PairNew.tsx

```diff
@@ -30,7 +30,12 @@ const Next = ({
   const { t } = useTranslation();
 
   return (
-    <Button type="main" size="large" onPress={onNext}>
+    <Button
+      type="main"
+      size="large"
+      onPress={onNext}
+      testID="Onboarding - Pair New Nano"
```

Comment:
Won't this give you troubles? Ids shouldn't have spaces

Reply:
Fixed